CONTRIBUTING.md

```diff
@@ -29,6 +31,16 @@ Console](https://console.cloud.google.com/apis/library).
 
         npm test
 
+### Running the tests for a Pull Request
+
+When a Pull Request is opened, reopened, or has new commits pushed the tests will be run.
+
+If the tests for a sample change do not run, they can be triggered by adding the `actions:force-run` label.
```

Comment:
This only triggers a subset of tests right?

Reply:
Yes, this is for the sample tests, as distinguished from all validations/checks that are run.

Added a comment with a suggestion to clarify.